transport_versioned_docs/version-1.0.0/workflow-client-server-secure.md

```diff
@@ -0,0 +1,147 @@
+
+
+#### Generate the CA-signed certificate to use with the server
+Create now a private key for the server. 
+```shell
+openssl genrsa -out myGameServerPrivate.key 2048
+```
+From this private key, you can generate a certificate signing request (csr). 
+```shell
+openssl req -new -key myGameServerPrivate.key -out myGameServerCertificateSigningRequest.csr
+```
+You'll be prompted with the same questions as for generating the root certificate.
+The answers are no more important. Make sure to use a proper `common name` though as it might be useful in case you need to debug the certificate.
+
+Finally, using the different files generated, we can create the certificate file the server will use to authenticate itself : 
+```shell
+openssl.exe x509 -req -in myGameServerCertificateSigningRequest.csr -CA myGameClientCA.pem -CAkey clientPrivateKeyForRootCA.key -CAcreateserial -out myGameServerCertificate.crt -days 365 -sha256
+```
+You should have now generated a total of five files. Out of these, only will be used later on : 
+* The content of the `myGameClientCA.pem` file, will be used client side as the `pem` parameter.
+* On the server end, `myGameServerPrivate.key` file content will be used for the `rsaKey` parameter.
+* `myGameServerCertificate.crt` will be used by the `rsa` parameter on the server end. 
+
+### Boiler Plate file holding the secure parameters
+Create a `SecureParameters.cs` script file to hold your the certificates and the private key. Place it in the same folder as the minimal server and minimal client scripts.
+Start by enabling Unity TLS with : 
+```cs 
+#define ENABLE_MANAGED_UNITYTLS
+```
+and add the following dependencies: 
+```cs
+using Unity.Collections;
+using Unity.Networking.Transport.TLS;
+```
+Then declare the secureParameters class and the boilerplate code that will hold your secure information:
+```cs
+public static class SecureParameters
+{
+        public static FixedString32Bytes commonName = new FixedString32Bytes("common-name");  // Use the common name you used to define the server certificate. 
+        public static FixedString4096Bytes myGameClientCA = new FixedString4096Bytes(
+        @"-----BEGIN CERTIFICATE-----
+                     ---                  
+          -----END CERTIFICATE-----"); // This should contain the content of myGameClientCA.pem 
+        public static FixedString4096Bytes myGameServerCertificate = new FixedString4096Bytes(
+        @"-----BEGIN CERTIFICATE-----
+                    ---
+          -----END CERTIFICATE-----"); // This should contain the content of myGameServerCertificate.pem  
+        public static FixedString4096Bytes myGameServerPrivate = new FixedString4096Bytes(
+        @"-----BEGIN RSA PRIVATE KEY-----
+                       ---
+          -----END RSA PRIVATE KEY-----"); // This should contain the content of myGameServerPrivate.key  
```

Comment:
Verbatim strings (`@` strings) should not be indented like the rest of the code. There should be no indentation, otherwise the indentation will make it in the string itself. Although here it's probably fine since certificate parsers will typically ignore whitespace. But still, better safe than sorry.

Reply:
Removed the indentation